<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetServer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frameTop">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>39</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="pbOpen">
          <property name="text">
           <string>&amp;Open</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pbClose">
          <property name="text">
           <string>C&amp;lose</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pbClear">
          <property name="text">
           <string>&amp;Clear</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="chkShowHexa">
          <property name="text">
           <string>Show Hexa</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="chkSendHexa">
          <property name="text">
           <string>Send Hexa</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabOption">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabTcp">
        <attribute name="title">
         <string>TCP</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0,0">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="lblTcpHost">
           <property name="text">
            <string>Host</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leTcpHost"/>
         </item>
         <item>
          <widget class="QLabel" name="lblTcpPort">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leTcpPort"/>
         </item>
         <item>
          <widget class="QToolButton" name="tbTcpAdvance">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabUdp">
        <attribute name="title">
         <string>UDP</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,0,0">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="lblUdpHost">
           <property name="text">
            <string>Host</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leUdpHost"/>
         </item>
         <item>
          <widget class="QLabel" name="lblUdpPort">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leUdpPort"/>
         </item>
         <item>
          <widget class="QToolButton" name="tbUdpAdvance">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabSsl">
        <attribute name="title">
         <string>SSL</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,0,0">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="lblSslHost">
           <property name="text">
            <string>Host</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSslHost"/>
         </item>
         <item>
          <widget class="QLabel" name="lblSslPort">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSslPort"/>
         </item>
         <item>
          <widget class="QToolButton" name="tbSslAdvanced">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPlainTextEdit" name="pteRecv">
        <property name="font">
         <font>
          <family>Courier New</family>
         </font>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPlainTextEdit" name="pteSend">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSend">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pbOpen</tabstop>
  <tabstop>pbClose</tabstop>
  <tabstop>pbClear</tabstop>
  <tabstop>chkShowHexa</tabstop>
  <tabstop>chkSendHexa</tabstop>
  <tabstop>leTcpHost</tabstop>
  <tabstop>leTcpPort</tabstop>
  <tabstop>tbTcpAdvance</tabstop>
  <tabstop>leUdpHost</tabstop>
  <tabstop>leUdpPort</tabstop>
  <tabstop>tbUdpAdvance</tabstop>
  <tabstop>leSslHost</tabstop>
  <tabstop>leSslPort</tabstop>
  <tabstop>tbSslAdvanced</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
